tests/mir-opt/copy-prop/dead_stores_better.rs

```diff
@@ -1,17 +1,24 @@
-// skip-filecheck
 // EMIT_MIR_FOR_EACH_PANIC_STRATEGY
 // This is a copy of the `dead_stores_79191` test, except that we turn on DSE. This demonstrates
 // that that pass enables this one to do more optimizations.
 
 //@ test-mir-pass: CopyProp
-//@ compile-flags: -Zmir-enable-passes=+DeadStoreElimination
+//@ compile-flags: -Zmir-enable-passes=+DeadStoreElimination-initial
```

Comment:
Currious why the change here? The CHECK statements look identical to those in `dead_stores_79191.rs` which makes the comment above outdated.

Reply:
Thanks! Now I confirmed that mir files from those tests are identical.
They were introduced in https://github.com/rust-lang/rust/pull/106908, but in the PR the `.f.CopyProp.after.mir` files already looked identical.
Should we remove this test as duplicated test?